CHANGELOG.md

```diff
@@ -1,3 +1,87 @@
+# v 1.12.0 (2022-01-24)
+
+## OS Changes
+
+* Disable strict aliasing for c-utf-8 library strict aliasing in dbus-broker ([#2730])
+* Add `/sys/firmware` to privileged mounts in host-ctr ([#2714])
+* Use user-provided registry credentials for public.ecr.aws in host-ctr ([#2676])
+* Build masked paths list dynamically in host-ctr ([#2637])
+* Enable EFI option in systemd ([#2714])
+* Allow simple enums as map keys in datastore ([#2687])
+* Add method for fetching hostname in imdsclient ([#2647])
```

Comment:
I might modify this (or add another item) to call out the stated goal of [the PR this was attached to](https://github.com/bottlerocket-os/bottlerocket/pull/2647):

e.g. "Improve reliability of `settings.network.hostname` generator"